core/store/migrations/migration1608217193/migrate.go

```diff
@@ -0,0 +1,9 @@
+package migration1608217193
+
+import "github.com/jinzhu/gorm"
+
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+		ALTER TABLE pipeline_specs ADD COLUMN max_task_duration bigint;
```

Comment:
Ah I see this got added here.

Reply:
Should it be nullable?

Reply:
Yeah I think it makes sense to keep it nullable to keep 0 open as a value (although if we wanted to actually support that we'd need to modify the Scan/Value for Interval or just use a pointer). Also we keep it nullable it other similar places like the OCR timeouts